tokio/src/runtime/builder.rs

```diff
@@ -233,6 +233,8 @@ impl Builder {
     /// # Panic
     ///
     /// This will panic if `val` is not larger than `0`.
+    // Alias for old name in 0.x
+    #[cfg_attr(docsrs, doc(alias = "max_threads"))]
```

Comment:
It's worth noting that, strictly speaking, `max_blocking_threads` does not do the same as `max_threads`.

Reply:
I can't find anything about the semantic change in the changelog or the PR https://github.com/tokio-rs/tokio/pull/3287

Reply:
Previously `max_threads` limited both blocking and worker threads, but the new `max_blocking_thread` does not include async worker threads in the count.